internal/controller/services/auth/auth_controller_actions.go

```diff
@@ -55,6 +55,12 @@ func initialize(ctx context.Context, rr *odhtypes.ReconciliationRequest) error {
 }
 
 func bindRole(ctx context.Context, rr *odhtypes.ReconciliationRequest, groups []string, roleBindingName string, roleName string) error {
+	namespace, err := actions.ApplicationNamespace(ctx, rr)
```

Comment:
In cases where services uses applicationNamespace, what happens if namespace changes in DSCI? Shouldn't services which needs application namespace watch DSCI?